core/identity-management/iin-agent/makefile

```diff
@@ -27,7 +27,7 @@ weaver-fabric-interop-sdk:
 	mkdir -p weaver-fabric-interop-sdk && \
 		cp -r ../../../sdks/fabric/interoperation-node-sdk/build ./weaver-fabric-interop-sdk/ && \
 		cp -r ../../../sdks/fabric/interoperation-node-sdk/index.js ./weaver-fabric-interop-sdk/ && \
-		cp -r ../../../sdks/fabric/interoperation-node-sdk/package-local.json ./weaver-fabric-interop-sdk/package.json
```

Comment:
Does this change work fine with the local build targets?

Reply:
In the driver we have this way only (whatever is after change).